src/processing/mode/android/AndroidBuild.java

```diff
@@ -179,6 +181,8 @@ public File createProject() throws IOException, SketchException {
 //      InputStream input = PApplet.createInput(getCoreZipLocation());
 //      PApplet.saveStream(new File(libsFolder, "processing-core.jar"), input);
       Base.copyFile(coreZipFile, new File(libsFolder, "processing-core.jar"));
+      
+      copySupportV4(libsFolder);
```

Comment:
I understand why it's *used*, I asked why it was being added to the repo. In this particular section of code, the version installed with the Android SDK is used, not the binary JAR that was added earlier. Does `AndroidMode` actually need this in its `CLASSPATH`, or is it needed during compile time for Android sketches. I suspect it's the latter. 

The Apache License is not compatible with the license already on this code. It also means we have to keep checking binary blobs into the repo whenever there's an update. And if we don't, we'll get conflicts with what's installed. It's a losing battle.

But even so, this might even be (another) reason to drop Gingerbread and go to API 15. What do you think @codeanticode?